app/models/user.rb

```diff
@@ -470,6 +470,11 @@ def coauthored_works
     return @coauthored_works
   end
 
+  #  Returns array of works where the user is the sole author
```

Comment:
I think this should be
```suggestion
  #  Returns array of collections where the user is the sole author
```
(@sarken please correct me if I got the wrong spot!)

Reply:
Ah geez. No you're exactly right. Thanks, addressed.